shell/src/main/java/alluxio/shell/command/CopyFromLocalCommand.java

```diff
@@ -230,7 +230,7 @@ private void copyPath(File src, AlluxioURI dstPath) throws IOException {
             buf.flip();
             os.write(buf.array(), 0, buf.limit());
           }
-        } catch (IOException e) {
+        } catch (IOException | RuntimeException e) {
```

Comment:
It will be easier to open a new PR. In the new PR, please only include the changes to `CopyFromLocalCommand.java`